sdk/monitor/opentelemetry-exporter-azuremonitor/src/main/java/com/azure/opentelemetry/exporter/azuremonitor/AzureMonitorExporter.java

```diff
@@ -0,0 +1,784 @@
+
+        if (inProc) {
+            remoteDependencyData.setType("InProc");
+        } else {
+            if (attributes.containsKey("http.method")) {
+                applyHttpRequestSpan(attributes, remoteDependencyData);
+            } else if (attributes.containsKey(SemanticAttributes.DB_SYSTEM.key())) {
+                applyDatabaseQuerySpan(attributes, remoteDependencyData, stdComponent);
+            } else if (span.getName().equals("EventHubs.send")) {
+                // TODO eventhubs should use CLIENT instead of PRODUCER
+                // TODO eventhubs should add links to messages?
+                remoteDependencyData.setType("Microsoft.EventHub");
+                String peerAddress = removeAttributeString(attributes, "peer.address");
+                String destination = removeAttributeString(attributes, "message_bus.destination");
+                remoteDependencyData.setTarget(peerAddress + "/" + destination);
+            } else if (span.getName().equals("EventHubs.message")) {
+                // TODO eventhubs should populate peer.address and message_bus.destination
+                String peerAddress = removeAttributeString(attributes, "peer.address");
+                String destination = removeAttributeString(attributes, "message_bus.destination");
+                if (peerAddress != null) {
+                    remoteDependencyData.setTarget(peerAddress + "/" + destination);
+                }
+                remoteDependencyData.setType("Microsoft.EventHub");
+            } else if ("kafka-clients".equals(stdComponent)) {
+                remoteDependencyData.setType("Kafka");
+                remoteDependencyData.setTarget(span.getName()); // destination queue name
+            } else if ("jms".equals(stdComponent)) {
+                remoteDependencyData.setType("JMS");
+                remoteDependencyData.setTarget(span.getName()); // destination queue name
+            }
+        }
+
+        remoteDependencyData.setId(span.getSpanId().toLowerBase16());
+        telemetryItem.getTags().put(ContextTagKeys.AI_OPERATION_ID.toString(), span.getTraceId().toLowerBase16());
+
+        SpanId parentSpanId = span.getParentSpanId();
+        if (parentSpanId.isValid()) {
+            telemetryItem.getTags().put(ContextTagKeys.AI_OPERATION_PARENT_ID.toString(), parentSpanId.toLowerBase16());
+        }
+
+        telemetryItem.setTime(getFormattedTime(span.getStartEpochNanos()));
+        remoteDependencyData.setDuration(getFormattedDuration(Duration.ofNanos(span.getEndEpochNanos() - span.getStartEpochNanos())));
+
+        remoteDependencyData.setSuccess(span.getStatus().isOk());
+        String description = span.getStatus().getDescription();
+        if (description != null) {
+            remoteDependencyData.getProperties().put("statusDescription", description);
+        }
+
+        Double samplingPercentage = removeAiSamplingPercentage(attributes);
```

Comment:
TODO for GA, probably need to publish an OpenTelemetry Sampler as part of this package that implements ApplicationInsights's sampling algorithm